R/outbreak_model.R

```diff
@@ -35,13 +35,13 @@
 #'                            k = 1.95,
 #'                            quarantine = FALSE)
 #'}
-outbreak_model <- function(num.initial.cases = NULL, prop.ascertain = NULL,
-                           cap_max_days = NULL, cap_cases = NULL,
-                           r0isolated = NULL, r0community = NULL,
-                           disp.iso = NULL, disp.com = NULL,
-                           k = NULL, delay_shape = NULL,
-                           delay_scale = NULL, prop.asym = NULL,
-                           quarantine = NULL) {
+outbreak_model <- function(num.initial.cases, prop.ascertainL,
```

Comment:
We use the NULLs to stop functions calling things from the global environment. This helps give clarity over if the function is working. Please put back. 

Also prop.ascertainL -> prop.ascertain

Reply:
I don't want to take up your time so I will put this back as requested.

However, I don't see the logic here. If there is no default, the function won't call anything from the global environment, it will just give an informative error.

```
f <- function(x) x ^ 2
x <- 4
f()
```

I guess if you mean when you're running the code line by line to debug and you always start by setting the parameters to default?

Reply:
Yes that is it :smile: I am not clear what the upside is from not setting it?

Reply:
The upside is that if you run a function without a needed argument, it immediately errors and tells you which argument you missed. Rather than trying to run the code with the arg set as NULL and seeing what happens.

Here's a contrived example just to be clear. The second function is doesn't error and probably isn't doing what you want it to do.